docs/notebooks/PCMSolvent.ipynb

```diff
@@ -1,36 +1,122 @@
 {
```

Comment:
<p>\begin{align}&nbsp;C(\vec{r}) = \begin{cases}0 &amp; \mathbf{r} \in \text{ inside cavity} \&nbsp;1 &amp; \mathbf{r} \in \text{ outside cavity}\end{cases}&nbsp;\end{align}&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MRChemSoft/vampyr/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>